website/admin.py

```diff
@@ -40,11 +40,15 @@ def get_admin_users_page(user):
         keyword_language = None if keyword_language == "null" else keyword_language
 
         filtering = False
-        if substring or start_date or end_date or language or keyword_language or category == "all":
+        if substring or start_date or end_date or language or keyword_language:
             filtering = True
 
         # After hitting 1k users, it'd be wise to add pagination.
```

Comment:
haha maybe this is the time now that we have more than 4000? Not a bad idea to implement (can be used in other pages also potentially like /explore?

Reply:
Good point! Will do this right away.